lib/index.js

```diff
@@ -1,3 +1,6 @@
+// used to avoid exporting same module
```

Comment:
Put comments on same line and use capitalization. Put this variable declaration below the imports please, near where it's used.